### ISSUE TITLE
PyPy: cannot find libpython

### ISSUE THREAD

**Issue by bgermann** [scikit-build/scikit-build#357]
The current way of finding libpython is insufficient on PyPy (Linux). `$ pypy3 -m sysconfig` gives
```
Platform: "linux-x86_64"
Python version: "3.5"
Current installation scheme: "pypy"

Paths: 
	data = "/opt/pypy3"
	include = "/opt/pypy3/include"
	platinclude = "/opt/pypy3/include"
	platlib = "/opt/pypy3/site-packages"
	platstdlib = "/opt/pypy3/lib-python"
	purelib = "/opt/pypy3/site-packages"
	scripts = "/opt/pypy3/bin"
	stdlib = "/opt/pypy3/lib-python"

Variables: 
	EXT_SUFFIX = ".pypy3-60-x86_64-linux-gnu.so"
	INCLUDEPY = "/opt/pypy3/include"
	SHLIB_SUFFIX = ".pypy3-60-x86_64-linux-gnu.so"
	SO = ".pypy3-60-x86_64-linux-gnu.so"
	SOABI = "pypy3-60"
	abiflags = ""
	base = "/opt/pypy3"
	exec_prefix = "/opt/pypy3"
	installed_base = "/opt/pypy3"
	installed_platbase = "/opt/pypy3"
	platbase = "/opt/pypy3"
	prefix = "/opt/pypy3"
	projectbase = "/opt/pypy3/bin"
	py_version = "3.5.3"
	py_version_nodot = "35"
	py_version_short = "3.5"
	srcdir = "/opt/pypy3/lib-python/config-3.5"
	userbase = "/home/user/.local"
```

https://github.com/scikit-build/scikit-build/blob/4a2eaf2ab67ff2e26331b2e991c8112885be491a/skbuild/cmaker.py#L400

**Comment by bgermann** [comment]
The libpypy3-c.so is in /opt/pypy3/bin in this case. I do not know the right way to locate it.

**Comment by jcfr** [comment]
See https://github.com/scikit-build/scikit-build/pull/407

**Comment by mattip** [comment]
#407 is closed, #418 is merged, can this be closed too?

**Comment by jcfr** [comment]
Indeed, thanks for the note :+1:
